### ISSUE TITLE
The problem with updating filters

### ISSUE THREAD

**Issue by Dizerma** [AdguardTeam/ContentBlocker#370]
UPD: Resolved. Решено

### Steps to reproduce

1. Open the application
2. Click on the update filters
3. Get an update error notification

### Expected behavior
 Updates should be checked and a message should be displayed stating that all the latest updates have been installed, or some filters have been successfully updated

### Actual behavior
 Update error. Please try again later

<details><summary>Screenshot:</summary>

 
![Screenshot_2024-04-24-21-58-45-71_ce5726043c1be52fab4b7ec464b079ef](https://github.com/AdguardTeam/ContentBlocker/assets/168023579/b48ee674-eea5-4cff-9160-58fea16bbe3f)

https://github.com/AdguardTeam/ContentBlocker/assets/168023579/bac66f61-baea-416d-8049-272f6efd6804

</details>

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| Adguard Content Blocker version:| 2.7.3
| Browser in use:| only Yandex Browser
| Filters enabled in Adguard:| The basic Adguard Filter, filter for counters and analytics systems, social media widget filter, irritant filter, Russian filter
| Device model:| Realme 10 pro plus
| Operating system and version:| Android 13
| Rooted or not? | No

**Comment by ameshkov** [comment]
Is it still happening?

**Comment by Dizerma** [comment]
> Is it still happening?

Yes

**Comment by ameshkov** [comment]
Давайте на русский перейдем.

Попробуйте пожалуйста в браузере открыть `filters.adtidy.org` и посмотреть все ли нормально открывается.

Если не нормально, то как именно выглядит ошибка в браузере?
Ну и мне нужно будет знать какой у вас провайдер/город.

**Comment by Dizerma** [comment]
> Давайте на русский перейдем.
> 
> Попробуйте пожалуйста в браузере открыть `filters.adtidy.org` и посмотреть все ли нормально открывается.
> 
> Если не нормально, то как именно выглядит ошибка в браузере? Ну и мне нужно будет знать какой у вас провайдер/город.

Открыл: 
![photo_2024-04-25_17-39-11](https://github.com/AdguardTeam/ContentBlocker/assets/168023579/027d3b1a-beac-460b-9384-b76877d8d710)

Как видно, ошибка 404.
Провайдеры:
Wi-Fi провайдер: Ростелеком.
Провайдер мобильного интернета: Tele2.
Проблема возникает через любое интернет-соединение (Wi-Fi и 4G). Город Иваново

**Comment by ameshkov** [comment]
Так, ну сервер доступен, значит дело не в этом. Разбираемся, спасибо

**Comment by Versty** [comment]
@Dizerma We have tried to reproduce this issue on our side, but to no avail. 
Would you mind reinstalling the app and let us know the result?

**Comment by Dizerma** [comment]
> @Dizerma We have tried to reproduce this issue on our side, but to no avail. Would you mind reinstalling the app and let us know the result?

I reinstalled it, but to no avail

**Comment by Dizerma** [comment]
Maybe it's possible to collect some traces, logs?

**Comment by Dizerma** [comment]
> @Dizerma We have tried to reproduce this issue on our side, but to no avail. Would you mind reinstalling the app and let us know the result?

Так я сообщил вам результат переустановки, что дальше делать? Проблема никуда не исчезла

**Comment by 2ahl55n061** [comment]
> > @Dizerma We have tried to reproduce this issue on our side, but to no avail. Would you mind reinstalling the app and let us know the result?
> 
> I reinstalled it, but to no avail

Same problem here, except I'm italian. Running the latest stable build of samsung internet browser, and content blocker app. No way I can retrieve downloads. Already reinstalled, saw no differences. Also, I'd like to report several of the filters proposed in the app are no longer supported, you better off removing them already, tbh.

**Comment by Dizerma** [comment]
> ### Действия по воспроизведению
> 
>     1. Откройте приложение
> 
>     2. Нажмите на фильтры обновления.
> 
>     3. Получить уведомление об ошибке обновления
> 
> 
> ### Ожидаемое поведение
> Необходимо проверить обновления и отобразить сообщение о том, что все последние обновления установлены или некоторые фильтры успешно обновлены.
> ### Фактическое поведение
> Ошибка обновления. Пожалуйста, повторите попытку позже
> Скриншот:
> ### Ваше окружение
> Описание 	Ценить
> Версия блокировщика контента Adguard: 	2.7.3
> Используемый браузер: 	только яндекс браузер
> Фильтры, включенные в Adguard: 	Базовый фильтр Adguard, фильтр для счетчиков и систем аналитики, фильтр виджетов соцсетей, фильтр раздражителей, русский фильтр
> Модель устройства: 	Реалме 10 про плюс
> Операционная система и версия: 	Андроид 13
> Укоренен или нет? 	Нет

ENG: Good news: the problem is solved. It turns out that I had some system trusted certificates turned off in the credential store (settings -> security & privacy -> additional security & privacy -> encryption & credentials). I just had to turn them all back on, and the update download will happen without errors. If the support staff had told me about the certificate store right away, the problem could have been fixed quickly, so keep that in mind for the future, support staff. 

RU: Сообщаю радостную новость: проблема решена. Оказывается, у меня были выключены некоторые системные доверенные сертификаты в хранилище учётных данных (настройки -> безопасность & конфиденциальность -> дополнительная безопасность & конфиденциальность -> шифрование & учётные данные). Нужно было просто включить их все обратно, и загрузка обновлений будет происходить без ошибок. Если бы сотрудники поддержки мне сразу сказали про хранилище сертификатов, проблему можно было бы быстро решить, так что имейте это в виду на будущее, сотрудники поддержки.
